.drone/drone.yml

```diff
@@ -1,182 +1,216 @@
 ---
-kind: pipeline
-name: prelude
-
-platform:
-  os: linux
-  arch: amd64
-
-steps:
-- name: make -B .drone/drone.yml
-  image: grafana/prometheus-pulsar-remote-write-build-image:c1b1dc1
-  commands:
-  - make -B .drone/drone.yml
-  - git diff --exit-code
-
+{
```

Comment:
Did something change in jsonnet that the output is mostly JSON formatted?

Reply:
I assume it's because of https://github.com/grafana/prometheus-pulsar-remote-write/pull/43/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R58 and the removal of the `--format` option. Is this correct?